specification.yaml

```
@@ -538,3 +667,52 @@ components:
             ## Subtitle
             
             ** Some Bold Text **
+            
+    LogEntry:
+      type: object
+      required: 
+        - entry_number
+        - content_type
+        - content_id 
+        - change_type 
+      properties:
+        entry_number:
+          type: number
+          example: 1
+        content_type:
+          type: string 
+          example: "POST"
+        content_id:
+          type: string 
+          example: "5ab3acce-e9d1-4b3a-be97-60d2cbe32a4c"
+        change_type:
+          type: string
+          example: "CREATED"
+          
+    CommunityRequest:
+      type: object
+      required: 
+        - request_type
+        - request 
+      properties:
+        request_type:
+          type: string 
+          example: "LOG_REQUEST"
+        request:
+          type: object
+          example:
+            {log_action: "STOP"}
+            
+    Event:
+      type: object 
+      required:
+        - event_type
+        - content
+      properties:
+        event_type:
```

```suggestion
        eventType:
```

We use camelCase everywhere else.